index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anthony Martin Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="header-logo">Anthony Martin</div>
+            <div class="header-menu-button">
+                <div></div>
+                <div></div>
+                <div></div>
+            </div>
+        </header>
+
+        <section class="headline">
+            <h1 class="headline_title">I’m Anthony
+                Glad to see you!</h1>
+            <p class="headline_description">I’m a software developer!
+                I can help you build a product , feature or website Look
+                through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t
+                hestiate to contact me.</p>
+            <a href="https://wa.me/584142221022">Let’s connect</a>
+            <nav>
+                <ul class="headline_social">
+                    <li><a href="#"><img src="img/social1.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="img/social2.svg" alt="LinkedIn"></a></li>
+                    <li><a href="#"><img src="img/social3.svg" alt="Social M"></a></li>
+                    <li><a href="#"><img src="img/social4.svg" alt="Github"></a></li>
+                    <li><a href="#"><img src="img/social5.svg" alt="Another Social Media"></a></li>
+                </ul>
+            </nav>
+        </section>
+        <section class="testing-buttons">
+            <button> See more!</button>
+            <button disabled> Disabled!</button>
+        </section>
+    </main>
+</body>
+
+</html>
```

Comment:
- Good job done so far, but there are some little differences between your design and the given Figma design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. This section of the design has a bottom radius at the bottom left-hand side which is missing in your design, please fix this 👍 👍 

Figma Design:
<img width="144" alt="egnengn" src="https://user-images.githubusercontent.com/73607512/204794162-fd43c1e4-64fd-4522-ac8a-d8971835889b.PNG">

Your project:
<img width="143" alt="htjedtk" src="https://user-images.githubusercontent.com/73607512/204794295-0fc5e2b8-63c5-4907-9f5e-4ed25d091834.PNG">